src/main/resources/db/migration/V1_0__create_initial_schema.sql

```diff
@@ -0,0 +1,113 @@
+create table attachment
+(
+    file_size          integer,
+    created            datetime(6),
+    attachment_data_id varchar(255) not null,
+    file_name          varchar(255),
+    id                 varchar(255) not null,
+    message_id         varchar(255) not null,
+    mime_type          varchar(255),
+    primary key (id)
+) engine = InnoDB;
+
+create table attachment_data
+(
+    id   varchar(255) not null,
+    file longblob,
+    primary key (id)
+) engine = InnoDB;
+
+create table conversation
+(
+    channel_id      varchar(255),
+    id              varchar(255) not null,
+    municipality_id varchar(255),
+    namespace       varchar(255),
+    topic           varchar(255),
+    primary key (id)
+) engine = InnoDB;
+
+create table conversation_meta_data
+(
+    conversation_id varchar(255) not null,
+    id              varchar(255) not null,
+    `key`           varchar(255),
+    meta_data_id    varchar(255) not null,
+    primary key (id)
+) engine = InnoDB;
+
+create table conversation_meta_data_values
+(
+    conversation_meta_data_id varchar(255) not null,
+    `values`                  varchar(255)
+) engine = InnoDB;
+
+create table conversation_participants
+(
+    conversation_id varchar(255) not null,
+    type            varchar(255),
+    value           varchar(255)
+) engine = InnoDB;
+
+create table message
+(
+    created         datetime(6),
+    content         varchar(255),
+    conversation_id varchar(255) not null,
+    id              varchar(255) not null,
+    in_reply_to     varchar(255),
+    sequence_number varchar(255),
+    type            varchar(255),
```

Comment:
I can't finf "type" in MessageEntity? What is this?

(we will need a `type` in conversation though. Move this to ConversationEntity?)